flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/ParquetTableSource.java

```diff
@@ -223,7 +224,8 @@ public TableSchema getTableSchema() {
 	@Override
 	public String explainSource() {
 		return "ParquetFile[path=" + path + ", schema=" + parquetSchema + ", filter=" + predicateString()
-			+ ", typeInfo=" + typeInfo + "]";
+			+ ", typeInfo=" + typeInfo + ", selectedFields=" + Arrays.toString(selectedFields)
```

Comment:
same here.

Reply:
`filter` and `predicates` are duplicate ?